src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ViewVersionSteps.java

```diff
@@ -0,0 +1,38 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import gov.healthit.chpl.aqa.pageObjects.ViewVersionPage;
+
+/**
+ * Class ViewVersionSteps definition.
+ */
+public class ViewVersionSteps extends Base {
+    /**
+     * Get user to the View Version Page.
+     * @param productVersionNumber is used to navigate to the view version page of that product
+     **/
+
+    @Given("^I navigate to edit version page of listing \"(.*)\"$")
+    public void editVersionPage(final String productVersionNumber) {
+        getDriver().get(getUrl() + "#/organizations/developers/1816/products/1742/versions/" + productVersionNumber);
```

Comment:
These IDs should be passed in as variables

Reply:
Agree with parametrizing IDs